src/assistant/Conversations/Sources/SourcesItem.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+import { useClient, generateWebLink } from 'cozy-client'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import ListItem from 'cozy-ui/transpiled/react/ListItem'
+import ListItemIcon from 'cozy-ui/transpiled/react/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+
+import { getDriveMimeTypeIcon } from 'assistant/Search/getIconForSearchResult'
+
+import styles from './styles.styl'
+
+const SourcesItem = ({ fileWitPath }) => {
+  const client = useClient()
+
+  const docUrl = generateWebLink({
+    slug: 'drive',
+    cozyUrl: client?.getStackClient().uri,
+    subDomainType: client?.getInstanceOptions().subdomain,
+    hash: `/folder/${fileWitPath.dir_id}/file/${fileWitPath._id}`
+  })
```

Comment:
I don't remember if we talked about this this morning, but this code won't work if the source is a note or a contact. Are those types possible?

Reply:
Oui on peut avoir des fichiers texte (notes, markdown, etc.) et des PDF